### PR TITLE
update error message when project is not passed

### PR DESCRIPTION
**Pull request by krishnaindani**
Resolved https://github.com/GoogleCloudPlatform/terraform-validator/issues/246

This updates the error message when the project is not passed.

### CODE REVIEW

**Comment by krishnaindani** [comment]
@googlebot I signed it!

File: converters/google/vendor_utils.go [hunk 1]
```diff
@@ -62,5 +62,5 @@ func getProjectFromSchema(projectSchemaField string, d converter.TerraformResour
 	if config.Project != "" {
 		return config.Project, nil
 	}
-	return "", fmt.Errorf("required field '%s' is not set", projectSchemaField)
+	return "", fmt.Errorf("required field '%s' is not set, you may use --project=my-project to provide default project to resolve the issue", projectSchemaField)
```
Comment (melinath) [inline converters/google/vendor_utils.go:65]: could this be: provide _a_ default project

**Review by melinath — CHANGES_REQUESTED** [review]
LGTM - just a small change request

File: converters/google/vendor_utils.go (hunk 1 above)
Comment (krishnaindani) [reply converters/google/vendor_utils.go:65]: so something simply like 
```suggestion
	return "", fmt.Errorf("required field '%s' is not set, provide a default project", projectSchemaField)
```

Comment (melinath) [reply converters/google/vendor_utils.go:65]: no, the version you had before was great, it was just missing the word "a" between `provide` and `default project`. So the full message would be:

```
required field '%s' is not set, you may use --project=my-project to provide a default project to resolve the issue
```

Comment (krishnaindani) [reply converters/google/vendor_utils.go:65]: oh okay, sorry I got that wrong. This is clear now, I will push the changes. Thanks

Comment (melinath) [reply converters/google/vendor_utils.go:65]: No worries! Thanks for making the update; I'm looking forward to incorporating this change!

Comment (krishnaindani) [reply converters/google/vendor_utils.go]: You're welcome. Updated the changes.

**Review by melinath — APPROVED** [review]
LGTM - thanks!

**Comment by krishnaindani** [comment]
> LGTM - thanks!

Thank you.

**Pull request merged**